src/options.asm

```diff
@@ -44,4 +44,166 @@
 
 ;ENABLE_DEBUG_3 equ 1
 
+IF(!DEF(VERSION))
+VERSION equs "0"
+ENDC
 
+FALSE equ 0
+TRUE equ 1
+
+BANK_1C_VAR = 1
+
+__PATCH_0__ = FALSE ; Applies to JP1+, EN1+, DE, FR
+__PATCH_1__ = FALSE ; Applies to DE, FR
+__PATCH_2__ = FALSE ; Applies to DE, JP1+, EN2+, FR
+__PATCH_3__ = FALSE ; Applies to JP1+, EN1+
+__PATCH_4__ = FALSE ; Applies to DE1+, JP2+, EN2+, FR1+
+__PATCH_5__ = FALSE ; Applies to DE
+__PATCH_6__ = FALSE ; Applies to JP
+__PATCH_7__ = FALSE ; Applies to FR
+__PATCH_8__ = FALSE ; Applies to JP1+
+__PATCH_9__ = FALSE ; Applies to DE, JP
+__PATCH_A__ = 0 ; 1 in JP, 2 in DE
+__PATCH_B__ = 0 ; 1 in JP/DE, 2 in FR
+__PATCH_C__ = FALSE ; Applies to EN
```

Comment:
I feel like these `__PATCH_XXX___` a bit confusing. The names don't really convey meaning, so I don't know how the patches relates to revisions.

Ideally, there would be some wa to give them meaning – like `__REVISION_2__`, or `LANG_FR`.

I'd say that:

- Probably the `_PATCH_XXX_` constants that only apply to a single language (like `__PATCH_7__`) can be converted to an `IF` right in the source code (like `IF LANG == FR`).
- Maybe some patches can be converted to a revision level instead.